### ISSUE TITLE
libstrophe doesn't filter forbidden sequences

### ISSUE THREAD

**Issue by pasis** [strophe/libstrophe#170]
More context:
 * https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=974205
 * https://github.com/profanity-im/profanity/issues/1437

**Comment by gbx-bfriedman** [comment]
We ran into this ourselves a few months ago while testing libstrophe with BLNS strings. It failed on the C0 control character range. It's a bit difficult to fix this on the application side since the surface area is anything touching libstrophe.

**Comment by jubalh** [comment]
[This](https://blns.io/) blns?

**Comment by gbx-bfriedman** [comment]
No, my apologies, I should have specified. [This](https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/blns.txt) BLNS.
